python/ray/dashboard/client/src/api.ts

```diff
@@ -109,14 +130,23 @@ export type NodeInfoResponse = {
 
 export const getNodeInfo = () => get<NodeInfoResponse>("/api/node_info", {});
 
+export type RayletCoreWorkerStats = {
```

Comment:
Is there a reason we need to distinguish between worker and core worker here? Seems like the abstraction from the worker code is leaking into this code.

Reply:
They're just fundamentally different types. We could change the way that we report stats from the dashboard to merge Raylet worker stats and Node-reporter worker stats before passing them to the dashboard, but that felt out of the scope of this PR.

The reason I broke it out into a separate type is because I think it is more readable to have it be a type with its own name, rather than the typescript type annotations referring to keys in a nested structure like I've seen elsewhere in the dashboard code.

Reply:
Sounds good. Might be good to unify them at a lower level in the future.